db/migrate/20180404100617_add_pay_scale_data.rb

```diff
@@ -0,0 +1,64 @@
+class AddPayScaleData < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
As is, if you rollback this migration will it leave the table untouched?

I have previously utilised both `up` and `down` to ensure the migration works in both directions.

Reply:
I'm not sure what the value of that in the specific migration is. So you think I should change this to up and add a down that removes all payscale data?